repolint.json

```diff
@@ -0,0 +1,375 @@
+{
+    "$schema": "https://raw.githubusercontent.com/todogroup/repolinter/master/rulesets/schema.json",
+    "version": 2,
+    "axioms": {
+      "linguist": "language",
+      "licensee": "license",
+      "packagers": "packager"
+    },
```

Comment:
Any reason to not extend the main config and just change what is needed? See example https://github.com/quic/.github?tab=readme-ov-file#customizing-repolinter-rules

Reply:
I just forgot you could do that